tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/PriceRepositoryTests.cs

```diff
@@ -63,11 +63,59 @@ public async Task GetPricesBySolutionIdQueryAsync_ValidSolutionId_ReturnsPriceLi
             request.Count().Should().Be(2);
         }
 
-        private static async Task InsertPriceAsync(string solutionId)
+        [Test]
```

Comment:
As per the comments in the repository, it shouldnt return a list, but a single result so these tests may need changing

Reply:
Discussed via call , this should return a list